### PR TITLE
23 resmio flow

### PR DESCRIPTION
**Pull request by menor**

### CODE REVIEW

**Comment by fabiennepimenta** [comment]
**Feedback:**
- > You should assign a `help request` label

 Great! 

- I like the colours colour scheme! The only one that is hard to find is the white one. 

- And I have a suggestion: When you push to test you should always write the Issue that you are solving (unless is an hotfix) and give a good title about the fix, so I can check CircleCI and see what's in coming or already on test – based on the title of the branch & issue number. 

</br>
I can validate this PR, but I think more people should take a look on this, specially because I'm not a developer. Or as we say in Portugal "speak now or forever hold your peace "  :) 

@resmio/developers @resmio/product

File: README.md [hunk 1]
```diff
@@ -96,6 +96,50 @@ color: forestGreen (#228B22)
...
+
+- **Thursday**: Review of issues still open / Estimation meeting
+- **Friday**: Sprint is compiled
+- **Monday**: Old Sprint ends / New Sprint starts ...
+
+## Personal Timeline
+
+### Which issue should you work on?
+1. If you have an opened Pull Request with required changes work on that first.
+2. Check if the are opened Pull Requests from your domain that need to be code reviewed and do that.
```
Comment (JannKleen) [inline README.md]: This is kind of obsolete now because they should always be assigned to someone, right?

File: README.md [hunk 2]
```diff
@@ -96,6 +96,50 @@ color: forestGreen (#228B22)
...
+3. If you are assigned to an issue start working on that one.
+4. If not check for issues labeled with your domain (eg: Backend) and `P1`.
```
Comment (JannKleen) [inline README.md]: We don't have those labels yet, maybe we should add them?

File: README.md (hunk 1 above)
Comment (fabiennepimenta) [inline README.md]: Yes, maybe we can change the sentence, but the logic is the same. 

`Check if there are any open Pull Request where you are assigned on to be code or product reviewed.`

and maybe an extra one


`Check if there are any open Pull Request where you are assigned on work on that` – something like this. For example, when someone asks for help or review, or if some of your work is missing, etc.

File: README.md (hunk 2 above)
Comment (menor) [inline README.md]: I was waiting for this (and the size and priority labels PR) to be merged, but feel free to create them if we need them now. Color is `#FF1493`

**Comment by menor** [comment]
@fabiennepimenta any suggestion about the color to replace the grey? (although that is not in theis PR)

About the commit, normally the branches are named as `issuenumber-issueshortdescription` (or at least they shoud)

**Pull request merged**